rnjane/bucket-list-app
# Integrate functionalities to features

What the PR does
Integrate impelemented functionalities to UI

How to test manually
clone the repository at https://github.com/rnjane/bucket-list-app
install requirements.txt
run app.py

Expected outcome
App runs, with crud features working correctly
